diff -Nur mhash-0.9.9-orig/include/mutils/mhash_config.h.in mhash-0.9.9/include/mutils/mhash_config.h.in
--- mhash-0.9.9-orig/include/mutils/mhash_config.h.in	2007-04-05 04:21:23.000000000 +0200
+++ mhash-0.9.9/include/mutils/mhash_config.h.in	2007-10-23 15:04:18.000000000 +0200
@@ -202,21 +202,6 @@
 /* Define to 1 if the C compiler supports function prototypes. */
 #undef PROTOTYPES
 
-/* The size of `unsigned char', as computed by sizeof. */
-#undef SIZEOF_UNSIGNED_CHAR
-
-/* The size of `unsigned int', as computed by sizeof. */
-#undef SIZEOF_UNSIGNED_INT
-
-/* The size of `unsigned long int', as computed by sizeof. */
-#undef SIZEOF_UNSIGNED_LONG_INT
-
-/* The size of `unsigned long long int', as computed by sizeof. */
-#undef SIZEOF_UNSIGNED_LONG_LONG_INT
-
-/* The size of `unsigned short int', as computed by sizeof. */
-#undef SIZEOF_UNSIGNED_SHORT_INT
-
 /* Define to 1 if you have the ANSI C header files. */
 #undef STDC_HEADERS
 
